### ISSUE TITLE
Hearts and Minds' first ability can lead to Ice wall having -1 advancement counter.

### ISSUE THREAD

**Issue by Larrea-Mon** [mtgred/netrunner#7519]
Hearts and Minds' first ability - Move an adv counter from a card you own to another card that can be advanced - seems to lack a stopgap for selecting cards with 0 advancement counters. It allowed me to select an Ice wall with 0 advancement counters on it, then move an advancement on it to an Agenda installed on that server. The resulting ice wall had 0 strength and no counters on it, at least jnet did not display -1 adv counters.

**Comment by NBKelly** [comment]
Duplicate of #7446
Will be closed by #7448, but we're currently waiting on deployment

**Comment by NBKelly** [comment]
Closes as completed via #7448
